src/components/views/elements/TruncatedList.js

```diff
@@ -28,17 +28,29 @@ module.exports = React.createClass({
         createOverflowElement: React.PropTypes.func
     },
 
+    getInitialState: function() {
```

Comment:
this looks unrelated... has it got lost?

I'm not quite sure what the intention is with it - it looks like what you are doing would be better handled by setting `truncateAt = -1`